app/src/main/res/values-ar/strings.xml

```diff
@@ -88,8 +88,9 @@
     <!-- FAQ -->
     <string name="faq_1_title">How can I fast-forward songs\?</string>
     <string name="faq_1_text">You can click on the current or max duration texts near the seekbar, that will move the song either backward, or forward.</string>
+    <string name="trackID">Track Number</string>
```

Comment:
add it at the proper place please, not at the bottom of the file

Reply:
Sorry, that was careless of me. I placed it under sorting for all the translations